.github/workflows/previous.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
   test-previous:
     strategy:
       matrix:
-        go-version: ['1.21.x', '1.22.x']
+        go-version: ['1.22.x']
```

Comment:
Not sure if we want to keep bumping this manually, but setup-go supports `'oldstable'` and `'stable'` [aliases](https://github.com/actions/setup-go?tab=readme-ov-file#using-stableoldstable-aliases) which they keep those up-to-date.